### PR TITLE
Replace @babel/polyfill with core-js and regenerator-runtime (updated, #11743)

### PR DESCRIPTION
**Pull request by alexandec**
## Description
`@babel/polyfill` has been deprecated as of 7.4.0. This PR does the following:

- Replace @babel/polyfill with `core-js/stable` and `regenerator-runtime/runtime` for apps, unit tests, and E2E tests
- Upgrade core-js from version 2 to version 3 (latest)
- Regenerate the list of polyfills in `src/platform/polyfills/preESModulesPolyfills.js` (used by older browsers, had to be re-created due to the core-js upgrade)
- Tell Babel to ignore `preESModulesPolyfills.js`, otherwise Babel will remove the imports from that file since they aren't needed for newer browsers. Fixes the issue we had in the first version of this PR: https://github.com/department-of-veterans-affairs/vets-website/pull/15282

## Testing done
Local smoke tests in IE 11 and Chrome, unit tests, E2E tests.

## Acceptance criteria
- [ ] App builds without errors and functions correctly
- [ ] All tests pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -25,13 +25,19 @@
...
         "useBuiltIns": "entry",
-        "corejs": "2.0.0",
+        "corejs": "3.8.1",
         "modules": false,
         "debug": false
       }
     ],
     "@babel/react"
   ],
+  "ignore": [
```
Comment (alexandec) [inline .babelrc:35]: This is the change compared to the original PR https://github.com/department-of-veterans-affairs/vets-website/pull/15282

Comment (U-DON) [reply .babelrc:35]: Looks another difference is that this the PR has no changes in `preESModulesPolyfills.js`. Am I correctly interpreting that having Babel ignore the file resulted in the same list of polyfills when you followed that process to regenerate it?

**Review by U-DON — APPROVED** [review]

File: .babelrc (hunk 1 above)
Comment (alexandec) [reply .babelrc:35]: This PR has a bunch of changes to preESModulesPolyfills.js, but GitHub is hiding those by default due to the large diff so it's easy to miss. The changes are small naming changes due to the core-js upgrade, and also the addition of new polyfills which were automatically added when regenerating. All the old polyfills are still present as well though

Comment (U-DON) [reply .babelrc:35]: Okay, I did miss that the first time around. I see it now. 👀

**Pull request merged**